openpype/hosts/nuke/api/lib.py

```diff
@@ -2224,15 +2208,14 @@ def get_write_node_template_attr(node):
     avalon_knob_data = read_avalon_data(node)
     # get template data
     nuke_imageio_writes = get_imageio_node_setting(
-        node_class=avalon_knob_data["family"],
+        node_class=avalon_knob_data["families"],
```

Comment:
Why is this `"families"`?

Reply:
it is actually used later as families, so why not to call it families from begginging.. There are 3 different subsets with main family sharing one `Write` family in families. It is for simplification of some publishing plugins and also for creator settings.
![image](https://user-images.githubusercontent.com/40640033/181581520-4294e810-3c41-4227-8bf3-272f6c0f8cab.png)